web/src/main/java/com/google/sps/data/Stock.java

```diff
@@ -18,12 +18,12 @@
 public final class Stock {
 
   private final long id;
-  private final String ticker;
-  private final long price;
+  private final String tik;
```

Comment:
You should keep this as ticker. Usually for readability purposes, we don't abbreviate because it will confuse other readers.